File: ZWORLDPAY/app/src/main/java/com/nechingu/zworldpay/util/ProgressUtil.java
package com.nechingu.zworldpay.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.nechingu.zworldpay.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2024/WellbeingLive/app/src/main/java/com/creator/wellbeinglive/util/ProgressUtil.kt
package com.creator.wellbeinglive.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.wellbeinglive.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}